src/seedu/addressbook/AddressBook.java

```diff
@@ -137,12 +137,9 @@
     private static final String DIVIDER = "===================================================";
 
 
-    /* We use a HashMap to store details of a person. Below are the keys to 
-     * store the respective data.
+    /* We use a HashMap to store details of a person. Use enum as keys.
      */
-    private static final String PERSON_PROPERTY_NAME = "name";
-    private static final String PERSON_PROPERTY_PHONE = "phone";
-    private static final String PERSON_PROPERTY_EMAIL = "email";
+    private enum PersonProperty {NAME, EMAIL, PHONE};
```

Comment:
Good to see that you have done the change I suggested for the previous commit :)